src/stdlib/builtins.md

```diff
@@ -226,12 +331,37 @@ Similarly, the following built-ins emit WebAssembly instructions accessing or ot
 * ```ts
   function load<T>(ptr: usize, immOffset?: usize): T
   ```
-  Loads a value of the specified type from memory. Equivalent to dereferencing a pointer in other languages.
+  <details><summary>Loads a value of the specified type from memory. Equivalent to dereferencing a pointer in other languages.</summary>
+
+  | T        | Instruction  | If context is i64
+  |----------|--------------|-------------------
+  | i8       | i32.load8_s  | i64.load8_s
+  | u8       | i32.load8_u  | i64.load8_u
+  | i16      | i32.load16_s | i64.load16_s
+  | u16      | i32.load16_u | i64.load16_u
+  | i32      | i32.load     | i64.load32_s
+  | u32      | i32.load     | i64.load32_u
+  | i64, u64 | i64.load     |
```

Comment:
```suggestion
  | i64, u64 |              |  i64.load
```

Reply:
If the type argument is `i64` or `u64`, it always emits an `i64.load`, independent of whether context is `i64`. Could add a `*same*` here, wdyt?

Reply:
`*n/a*` will be fine. Same for `i64.store` case